src/puppetlabs/puppetdb/query_eng/engine.clj

```diff
@@ -1214,13 +1255,20 @@
              :state state}
             :else {:node node :state state}))
 
+(pls/defn-validated op-to-string
```

Comment:
Looks like this function is just `(comp (partial map string/lower-case) utils/vectorize-maybe)`